README.md

```diff
@@ -250,6 +250,16 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Compatibility with other annotation processors
```

Comment:
This section is really about incompatibility with the `-processor` command-line argument.